public/pages/CreateTrigger/containers/DefineBucketLevelTrigger/DefineBucketLevelTrigger.js

```diff
@@ -309,7 +309,12 @@ class DefineBucketLevelTrigger extends Component {
         <div style={{ padding: '0px 10px', paddingTop: '10px' }}>
           <FormikFieldText
             name={`${fieldPath}name`}
-            fieldProps={{ validate: validateTriggerName(triggers, triggerValues, fieldPath) }}
```

Comment:
Can't we just update the inputs here and not expand it out?

Reply:
nope, the original func is incorrect and it even could not return the empty name validation as expected, the validate itself expect a value, and our `validateTriggerName` would return value per condition